Apps/phtaniumrest/readme.html

```diff
@@ -3,26 +3,87 @@
 
   Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
 -->
-<p><b>Run Query Guidelines</b>
+<h2>Asset Configuration</h2>
+<li><b>Consider question results complete at (% out of 100)</b></li>
+<ul>
+  <li>This asset parameter refers to the amount of question results you wish to be returned by actions <b>run query</b> and <b>list processes</b> before considering the action to be completed.
```

Comment:
Recommended phrasing:
Consider Tanium question results complete at this value, a percentage out of 100. This parameter impacts the **run query** and **list processes** actions only. Note that a similar value can be defined in Tanium user preferences – you might want to reflect the same value in your app asset configuration as you use in your Tanium user configuration